frontend/src/DataImporter/buildNetwork.test.ts

```diff
@@ -0,0 +1,322 @@
+import { Edge, Node, ChanelEdge, buildNetwork, groupByChanel, handleNode, handleEdge, mergeEdges, Network, Chanel, mergeSpectrum, checkEdgeExists, chanelNode } from "./buildNetwork";
+import { EdgeDataRow, NodeDataRow, EdgeSpectrumDataRow ,parseEdges, parseEdgeSpectrum , parseNodes} from "./parseCsv";
+
```

Comment:
dopiszę to jeszcze do README, ale mamy w projekcie skonfigurowane reguły lintera (są dosyć minimalistyczne, można je jeszcze uzupełnić, ale na razie zapewniają w miarę spójne formatowanie)

można sprawdzić naruszenia reguł przez `npm run lint`
do każdego IDE są wtyczki który będą automatycznie formatować kod na podstawie konfiguracji eslint
a ja jeszcze dodam do tego komendę

Reply:
poprawiłem co linter zgłaszał. były to tylko wcięcia. jeśli jest coś więcej to daj znać bo może wtedy nie czytać configa